foreman/data_refinery_foreman/surveyor/array_express.py

```diff
@@ -104,8 +105,8 @@ def create_experiment_from_api(self, experiment_accession_code: str) -> Dict:
             experiment_object.description = parsed_json["description"][0]["text"]
             experiment_object.platform_name = experiment["platform_accession_name"]
             experiment_object.platform_accession_code = experiment["platform_accession_code"]
-            experiment_object.source_first_published = experiment["release_date"]
-            experiment_object.source_last_updated = experiment["last_update_date"]
+            experiment_object.source_first_published = parse_datetime(experiment["release_date"])
```

Comment:
Quick little testerino might be nice here..

If it doesn't work, there is a library called `arrow` that I like for situations like this